### PR TITLE
[bitnami/elasticsearch] Add support for architecture [standalone or cluster]

### PR DESCRIPTION
**Pull request by fabiocastagnino**
### Description of the change

Added the option to deploy single-node cluster.
Like other Bitnami char, it user the value "architecture"
- cluster: for multi-node deployment (like before this change)
- standalone: for single-node deployment

### Benefits

Sometimes can be helpful the ability to deploy elasticsearch in single node architecture.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/elasticsearch/values.yaml [hunk 1]
```diff
@@ -208,6 +208,10 @@ image:
   ##
   debug: false
 
+## @param architecture Elasticsearch architecture (`standalone` or `cluster`)
+##
+architecture: cluster
```
Comment (andresbono) [inline bitnami/elasticsearch/values.yaml:213]: What if we add some extra context about the implications of setting `standalone` here?
- Only the `master` role configuration would be taken into account, the rest will be ignored: `data`, `coordinating`...
- `.master.replicaCount` will be ignored
- ...

**Review by andresbono — COMMENTED** [review]
Thank you very much for your contribution! I left some minor comments, but overall, LGTM. Could you take a look at those?

**Comment by fabiocastagnino** [comment]
> Thank you very much for your contribution! I left some minor comments, but overall, LGTM. Could you take a look at those?

Sure, tomorrow I'll push the update. Thanks for the suggestion

**Comment by andresbono** [comment]
Hi @fabiocastagnino, please allow us some more time before we continue with the review process. We are having second thoughts on merging this proposal as is. I will keep you posted.

**Comment by fabiocastagnino** [comment]
> Hi @fabiocastagnino, please allow us some more time before we continue with the review process. We are having second thoughts on merging this proposal as is. I will keep you posted.

Let me know if I can improve anything!

**Comment by andresbono** [comment]
@fabiocastagnino, we opened a new PR #19529 to address your use case (single-node cluster) but approaching it in a different way.

For instance, we will not alter the existing inputs interface. A single-node cluster is only a very particular scenario for the existing "cluster architecture". For that reason, it is not worth adding the concept of "architecture" in the chart.

Please, check the new PR and let us know your thoughts! If you agree, we will close this PR in favor of #19529.

**Comment by fabiocastagnino** [comment]
> @fabiocastagnino, we opened a new PR #19529 to address your use case (single-node cluster) but approaching it in a different way.
> 
> For instance, we will not alter the existing inputs interface. A single-node cluster is only a very particular scenario for the existing "cluster architecture". For that reason, it is not worth adding the concept of "architecture" in the chart.
> 
> Please, check the new PR and let us know your thoughts! If you agree, we will close this PR in favor of #19529.

Ok, I'm closing this PR

**Pull request closed without merging**